backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -11,10 +11,7 @@
 import gov.cdc.usds.simplereport.api.model.PersonUpdate;
 import gov.cdc.usds.simplereport.api.model.pxp.PxpRequestWrapper;
 import gov.cdc.usds.simplereport.api.model.pxp.PxpVerifyResponse;
-import gov.cdc.usds.simplereport.db.model.PatientLink;
-import gov.cdc.usds.simplereport.db.model.PatientPreferences;
-import gov.cdc.usds.simplereport.db.model.Person;
-import gov.cdc.usds.simplereport.db.model.TestEvent;
+import gov.cdc.usds.simplereport.db.model.*;
```

Comment:
I keep thinking we have a checkstyle rule saying not to import `*` but it keeps not being true. Please don't import `*`. 🙂

Reply:
Ah, sorry. IntelliJ defaults to `*` when there's >=5 imports from the same package. Will revert.